addons/mass_mailing_sale/models/mailing_mailing.py

```diff
@@ -15,47 +15,46 @@ class MassMailing(models.Model):
 
     @api.depends('mailing_domain')
     def _compute_sale_quotation_count(self):
-        has_so_access = self.env['sale.order'].check_access_rights('read', raise_exception=False)
-        if not has_so_access:
-            self.sale_quotation_count = 0
-            return
         for mass_mailing in self:
-            mass_mailing.sale_quotation_count = self.env['sale.order'].search_count(mass_mailing._get_sale_utm_domain())
+            mass_mailing.sale_quotation_count = self.env['sale.order'].sudo().search_count(mass_mailing._get_sale_utm_domain())
```

Comment:
I am wondering if we should not simply search on source, instead of the whole domain... maybe to check with Lna and Dbe to be sure we are consistent, but it could simplify a lot the computation while still being accurate.